lib/grape_entity/json.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+module Grape
+  class Entity
+    if defined?(::MultiJson)
+      Json = ::MultiJson
+    else
+      Json = ::JSON
+      Json::ParseError = Json::ParserError
```

Comment:
why this line?

Reply:
I took the code from grape directly. In fact, since grape-entity is not parsing, we could drop it.

Reply:
ok